### PR TITLE
Bt 15 memberships

### PR DESCRIPTION
**Pull request by harishshams**
Please review and approve

### CODE REVIEW

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/MembershipsPage.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    public void saveNewMemberships(String membershipName) {
+        click(By.cssSelector(membershipAddButtonClassName));
+        setText(By.cssSelector(membershipNameTextAreaCssSelector), membershipName);
+        if (getText(By.xpath(membershipNameAlreadyExistsLabel)).contains("Already exists")) {
+            click(By.xpath(addMembershipCancelButtonXPath));
+        } else {
+            click(By.xpath(membershipSaveButtonXPath));
+        }
+        isElementVisible(By.cssSelector(membershipTableClassName));
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/MembershipsPage.java]: see if line 39 to 45 are needed. Else remove

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/MembershipsTests.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+@Test
+public void addNewMembership() {
+    LoginPage loginPage = new LoginPage(driver);
+    loginPage.login("admin", "admin123", true, null);
+    HeaderPage headerPage = new HeaderPage(driver);
+    headerPage.selectMenu(MenuOptions.MEMBERSHIPS);
+    MembershipsPage membershipsPage = new MembershipsPage(driver);
+    membershipsPage.saveNewMemberships("Level1");
+    String uuid = UUID.randomUUID().toString();
+    membershipsPage.saveNewMemberships("level2 " + uuid);
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/MembershipsTests.java]: saveNewMembershipAndVerifyItsAdded

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/MembershipsPage.java (hunk 1 above)
Comment (harishshams) [reply src/test/java/ca/qaguru/oranghrmbatch27/pages/MembershipsPage.java]: Done

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/MembershipsTests.java (hunk 2 above)
Comment (harishshams) [reply src/test/java/ca/qaguru/oranghrmbatch27/tests/MembershipsTests.java]: Done

**Review by RahulNair92 — APPROVED** [review]
Verified and approved

**Pull request merged**